bundles/binding/org.openhab.binding.ipx800/src/main/java/org/openhab/binding/ipx800/internal/Ipx800Binding.java

```diff
@@ -0,0 +1,299 @@
+import org.openhab.binding.ipx800.internal.itemslot.Ipx800SimpleClic;
+
+import org.openhab.core.binding.AbstractBinding;
+import org.openhab.core.binding.BindingProvider;
+import org.openhab.core.types.Command;
+import org.openhab.core.types.State;
+import org.osgi.service.cm.ConfigurationException;
+import org.osgi.service.cm.ManagedService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The ipx800 binding. Handle the creation of items object linked to the devices.
+ * 
+ * @author Seebag
+ * @since 1.7.0
+ */
+public class Ipx800Binding extends AbstractBinding<Ipx800BindingProvider>
+		implements ManagedService {
+
+	private static final Logger logger = LoggerFactory
+			.getLogger(Ipx800Binding.class);
+	/**
+	 * Store the all ipx800 devices DeviceName, Ipx800DeviceConnector
+	 */
+	private Map<String, Ipx800DeviceConnector> devices = new HashMap<String, Ipx800DeviceConnector>();
+
+	public Ipx800Binding() {
+	}
+
+	@Override
+	public void bindingChanged(BindingProvider prov, String itemName) {
+		if (prov.getItemNames().contains(itemName)) {
+			try {
+				createItem(itemName, prov);
+			} catch (Ipx800UnknownDeviceException e) {
+				logger.error("Item {} will be ignored", itemName);
+			}
+		} else {
+			logger.debug("Removing item {}", itemName);
+			for (String deviceName : devices.keySet()) {
+				for (Ipx800Port slot : devices.get(deviceName).getAllPorts()) {
+					slot.destroyItem(itemName);
+				}
+			}
+		}
+	}
+	/**
+	 * Reload and reattach all items to all devices
+	 */
+	public void reloadItems() {
+		logger.trace("Size of providers : {}", providers.size());
+		for (Ipx800BindingProvider provider : providers) {
+			for (String itemName : provider.getItemNames()) {
+				try {
+					createItem(itemName, provider);
+				} catch (Ipx800UnknownDeviceException e) {
+					logger.error("Item {} will be ignored", itemName);
+				}
+			}
+		}
+	}
+	
+	private Tuple<Ipx800DeviceConnector, Ipx800Port> getDeviceAndPort(String name, String portString) throws Ipx800UnknownDeviceException {
+		if (name.isEmpty() || portString.isEmpty()) {
+			return new Tuple<Ipx800DeviceConnector, Ipx800Port>(null, null);
+		}
+		int extDelta = 0;
+		Ipx800DeviceConnector device = devices.get(name);
+		
+		if (device == null) {
+			// Try to find extensions
+			for(Ipx800DeviceConnector dev : devices.values()) {
+				extDelta = dev.getExtensionDelta(name);
+				if (extDelta != 0) {
+					device = dev;
+					break;
+				}
+			}
+			if (extDelta == 0) {
+				throw new Ipx800UnknownDeviceException("Device '" + name + "' doesn't exist, please check your config/items");
+			}
+		}
+		return new Tuple<Ipx800DeviceConnector, Ipx800Port>(device, device.getPort(portString, extDelta));
+	}
+	
+	/**
+	 * Create and attach new item to device
+	 * @param itemName 
+	 * @param provider
+	 * @throws Ipx800UnknownDeviceException 
+	 */
+	private void createItem(String itemName, BindingProvider prov) throws Ipx800UnknownDeviceException {
```

Comment:
since "Item" is a reserved word in openHAB i strongly recommend to rename this method